tests/tool/Integration.Tests/IntegrationScenarios/WpfSample/vb/Upgraded/WpfApp1/WpfApp1.vbproj

```diff
@@ -19,6 +19,14 @@
     <NoWarn>41999,42016,42017,42018,42019,42020,42021,42022,42032,42036,42314</NoWarn>
   </PropertyGroup>
   <ItemGroup>
+    <Import Include="System.Threading.Tasks" />
+    <Import Include="System.Linq" />
+    <Import Include="System.Xml.Linq" />
+    <Import Include="Microsoft.VisualBasic" />
+    <Import Include="System" />
+    <Import Include="System.Collections" />
+    <Import Include="System.Collections.Generic" />
+    <Import Include="System.Diagnostics" />
```

Comment:
I believe these fall into a category of "default imports" that used to be provided from the .net runtime template

Reply:
Looks like these have been added as global usings for *all* projects for [C# 10](https://anthonygiretti.com/2021/07/18/introducing-c-10-global-usings-example-with-asp-net-core-6/), but only when you target .net 6 [see here](https://github.com/dotnet/sdk/commit/85943958e5a01d6bbd39a947aec2a4a37b8ba7cf#diff-5dfdd31992820273a25b399353a85ec6de6bdd620424a8adab3238c6ffd5df38). These were already part of VB, but now aren't. I'll look into if that is a regression

Reply:
good eye. i missed that!